pkg/scannerv4/mappers/mappers.go

```diff
@@ -405,6 +458,17 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 			// The preferred CVSS metrics will always be stored at the first index.
 			preferredCVSS = metrics[0]
 		}
+
+		var vulnEPSS *epss.EPSSItem
+		if epssVulnItem, ok := epssItems[v.ID]; ok {
+			if v, ok := epssVulnItem[cve]; ok {
```

Comment:
we may want to consider adding a check for `foundCVE` here, too, but we can do that in a followup